lib/Runtime/Provisioner/Provisioner.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Runtime/Provisioner/Provisioner.h"
+#include "glow/Backends/BackendUtils.h"
+#include "glow/Backends/CompiledFunction.h"
+#include "glow/Graph/Graph.h"
+
+#include <future>
+#include <queue>
+
+using namespace glow;
+using namespace runtime;
+using DeviceID = unsigned int;
```

Comment:
RuntimeTypes.h has `using DeviceIDTy = size_t;`, is this device ID the same or something different?  If different, can you make the type names more distinctive?

Reply:
It's actually not used anywhere, so I'm removing it.